src/routes/profile.js

```diff
@@ -64,6 +68,24 @@ router.post('/mod/ajax/processavatarrequest', isModMiddleware, (req, res) => {
       if (req.body.decision === true || req.body.decision === 'true') {
         console.log(`search lower user: ${foundReq.forUsername.toLowerCase()}`);
 
+        const updateLink = async (link) => {
+          const index = link.indexOf('pending_avatars');
```

Comment:
Let's update this guy and conform to a nicer link schema:

`http[s]://<subdomain>.<domain>/<key>`

Reply:
Ideally we can separate based on the domain name, not on something dependent on the key itself.

Reply:
Thoughts on keeping the construction in s3Agent.ts as it is specific to that usecase